### ISSUE TITLE
Rotation Control

### ISSUE THREAD

**Issue by OccultSlolem** [Nuts-Bolts-FRC-7528/FRC-2020#7]
Parameters:

- If the FMS is _not_ distributing a color, than we know that we should be in rotation control, so we can use a single button for both rotation and position
- When gamepad button is pressed, the motor for the spinner will go a certain measured number of encoder ticks using closed-loop control

```
if(PositionControlColor.length() == 0) { //If FMS has NOT given a color for position control yet (ie it is not unlocked and we need to do rotation control first)
    RotationControl(); //Do rotation control
}
```

**Comment by OccultSlolem** [comment]
Awaiting further discussion on 2/3

**Comment by OccultSlolem** [comment]
Merged into #17
